tests/testthat/test-Rules-class.R

```diff
@@ -80,6 +80,22 @@ test_that("NextBestDualEndpoint object can be created with user constructor 2",
   expect_valid(result, "NextBestDualEndpoint")
 })
 
+## NextBestMinDist ----
+
+test_that("NextBestMinDist works as expected", {
+  result <- expect_silent(.NextBestMinDist(target = 0.2))
+  expect_valid(result, "NextBestMinDist")
+})
+
+test_that("NextBestMinDist object can be created with user constructor", {
+  result <- expect_silent(
+    NextBestMinDist(
+      target = 0.2
+    )
+  )
+  expect_valid(result, "NextBestMinDist")
```

Comment:
maybe also check that the target is 0.2

Reply:
Tx @danielinteractive ! - how can I check this?

Reply:
just something like `expect_equal(result@target, 0.2)` ?

Reply:
Ohh sure, I am sorry I though your comment is related to the new `prototype` with `target = 0.2` I added :)